ChangeLog.md

```diff
@@ -2874,273 +2874,273 @@
+  + **fencing:** Do not wait for the query timeout if all replies have arrived
+  + **fencing:** Fix passing of parameters from CMAN containing '='
+  + **fencing:** Fix non-comparison when sorting devices by priority
+  + **fencing:** On failure, only try a topology device once from the remote level.
+  + **fencing:** Only try peers for non-topology based operations once
+  + **fencing:** Retry stonith device for duration of action's timeout period.
+  + **heartbeat:** Remove incorrect assert during cluster connect
+  + **ipc:** Bug cl#5110 - Prevent 100% CPU usage when looking for synchronous replies
+  + **ipc:** Use 50k as the default compression threshold
+  + **legacy:** Prevent assertion failure on routing ais messages (bnc#805626)
+  + **legacy:** Re-enable logging from the pacemaker plugin
+  + **legacy:** Relax the 'active' check for plugin based clusters to avoid false negatives
+  + **legacy:** Skip peer process check if the process list is empty in `crm_is_corosync_peer_active()`
+  + **mcp:** Only define `HA_DEBUGLOG` to avoid agent calls to `ocf_log` printing everything twice
+  + **mcp:** Re-attach to existing pacemaker components when mcp fails
+  + **pengine:** Any location constraint for the slave role applies to all roles
+  + **pengine:** Avoid leaking memory when cleaning up failcounts and using containers
+  + **pengine:** Bug cl#5101 - Ensure stop order is preserved for partially active groups
+  + **pengine:** Bug cl#5140 - Allow set members to be stopped when the subseqent set has require-all=false
+  + **pengine:** Bug cl#5143 - Prevent shuffling of anonymous master/slave instances
+  + **pengine:** Bug rhbz#880249 - Ensure orphan masters are demoted before being stopped
+  + **pengine:** Bug rhbz#880249 - Teach the PE how to recover masters into primitives
+  + **pengine:** cl#5025 - Automatically clear failcount for start/monitor failures after resource parameters change
+  + **pengine:** cl#5099 - Probe operation uses the timeout value from the minimum interval monitor by default (#bnc776386)
+  + **pengine:** cl#5111 - When clone/master child rsc has on-fail=stop, insure all children stop on failure.
+  + **pengine:** cl#5142 - Do not delete orphaned children of an anonymous clone
+  + **pengine:** Correctly unpack active anonymous clones
+  + **pengine:** Ensure previous migrations are closed out before attempting another one
+  + **pengine:** Introducing the whitebox container resources feature
+  + **pengine:** Prevent double-free for cloned primitive from template
+  + **pengine:** Process `rsc_ticket` dependencies earlier for correctly allocating resources (bnc#802307)
+  + **pengine:** Remove special cases for fencing resources
+  + **pengine:** rhbz#902459 - Remove rsc node status for orphan resources
+  + **systemd:** Gracefully handle unexpected DBus return types
   + Replace the use of the insecure mktemp(3) with mkstemp(3)
 
 # Pacemaker-1.1.8 (20 Sep 2012)
```

Comment:
Everything under the Pacemaker-1.1.8 heading has addition spaces between lines - I think this is supposed to be `##` instead of `#`.